app/controller.js

```diff
@@ -109,6 +114,10 @@ class MainController {
                 return;
             }
 
+            const isMacMeta = window.navigator.platform === 'MacIntel' && e.metaKey
```

Comment:
I guess this is the right handling for "Ctrl" for mac users. If so, can you implement it on every "Ctrl" used here (for example, Ctrl+leftArrow).